docs/management/api/list-actions-api.asciidoc

```diff
@@ -0,0 +1,135 @@
+[[list-actions-api]]
+=== List response actions
+
+Retrieves a list of response actions.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/action`
+
+
+==== URL query parameters
+
+All parameters are optional:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description| Default
+
+|`page` |number |The page of results to retrieve. | 1
+|`pageSize` |number |Size of results per page. 1 to 10000. | 10
+|`commands` |string[] a|A list of response action command names.
+
+Accepted values are:
+
+* `isolate`
+* `unisolate`
+* `kill-process`
+* `suspend-process`
+* `running-processes`
+
+|
+|`agentIds` |string[] |A list of agent ids. Max of 50. |
+|`userIds` |string[] |A list of user ids. |
+|`startDate` |string |A start date in ISO format or {ref}/common-options.html#date-math[Date Math format]. |
+|`endDate` |string |A start date in ISO format or {ref}/common-options.html#date-math[Date Math format]. |
+
+|
+|==============================================
+
+
+===== Example requests
+
+List actions:
+
+[source,sh]
+--------------------------------------------------
+GET /api/endpoint/action
+--------------------------------------------------
+// KIBANA
+
+List actions in the last 24 hours for two specific agent ids and commands of either `isolate` or `kill-process`:
```

Comment:
Want to confirm what this code is saying. Is it listing all `isolate` or `kill-process` commands in the past 24 hours for two specific agent IDs? 

If so, my recommendation: 
```suggestion
Lists `isolate` or `kill-process` command actions in the last 24 hours for two specific agent IDs:
```

Sorry if I misunderstood!

Reply:
@jmikell821 - Yes, exactly right. Thanks for the re-wording 😄